src/main/java/mate/project/service/OrderService.java

```diff
@@ -0,0 +1,19 @@
+package mate.project.service;
+
+import java.util.List;
+import mate.project.dto.OrderItemResponseDto;
+import mate.project.dto.OrderResponseDto;
+import mate.project.dto.PlaceOrderDto;
+import mate.project.dto.StatusDto;
+
+public interface OrderService {
+    OrderResponseDto placeOrder(PlaceOrderDto placeOrderDto);
+
+    List<OrderResponseDto> findAll();
+
+    void update(Long id, StatusDto statusDto);
```

Comment:
The update method does not return any value. Consider returning a boolean or the updated object to indicate success or failure of the update operation.